catalogs/catalog-jdbc-mysql/src/main/java/com/datastrato/gravitino/catalog/mysql/MySQLCatalogOperations.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2024 Datastrato Pvt Ltd.
+ * This software is licensed under the Apache License version 2.
+ */
+package com.datastrato.gravitino.catalog.mysql;
+
+import com.datastrato.gravitino.catalog.jdbc.JdbcCatalogOperations;
+import com.datastrato.gravitino.catalog.jdbc.JdbcTablePropertiesMetadata;
+import com.datastrato.gravitino.catalog.jdbc.converter.JdbcColumnDefaultValueConverter;
+import com.datastrato.gravitino.catalog.jdbc.converter.JdbcExceptionConverter;
+import com.datastrato.gravitino.catalog.jdbc.converter.JdbcTypeConverter;
+import com.datastrato.gravitino.catalog.jdbc.operation.JdbcDatabaseOperations;
+import com.datastrato.gravitino.catalog.jdbc.operation.JdbcTableOperations;
+import java.sql.Driver;
+import java.sql.DriverManager;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class MySQLCatalogOperations extends JdbcCatalogOperations {
+  private static final Logger LOG = LoggerFactory.getLogger(MySQLCatalogOperations.class);
+
+  public MySQLCatalogOperations(
+      JdbcExceptionConverter exceptionConverter,
+      JdbcTypeConverter jdbcTypeConverter,
+      JdbcDatabaseOperations databaseOperation,
+      JdbcTableOperations tableOperation,
+      JdbcTablePropertiesMetadata jdbcTablePropertiesMetadata,
+      JdbcColumnDefaultValueConverter columnDefaultValueConverter) {
+    super(
+        exceptionConverter,
+        jdbcTypeConverter,
+        databaseOperation,
+        tableOperation,
+        jdbcTablePropertiesMetadata,
+        columnDefaultValueConverter);
+  }
+
+  @Override
+  public void close() {
+    super.close();
+
+    try {
+      // Close thread AbandonedConnectionCleanupThread
+      Class.forName("com.mysql.cj.jdbc.AbandonedConnectionCleanupThread")
```

Comment:
could you make this logic reusable?

Reply:
Not possible. I have tried to move it to the common module used by `catalog-iceberg` and `catalog-jdbc`, however,  since common module is not loaded by IsolatedClassloader, it does not work.

Reply:
ok